autorest/codegen/serializers/builder_serializer.py

```diff
@@ -567,7 +567,7 @@ def response_docstring(self, builder: OperationType) -> List[str]:
         return [
             response_str,
             rtype_str,
-            ":raises: ~azure.core.exceptions.HttpResponseError",
+            ":raises ~azure.core.exceptions.HttpResponseError: If there is error in response",
```

Comment:
ooh nice, so it was the colon placement

I don't think we need the "If there is error in response", since that's what `raises` describes itself. So I think this can just be `:raises ~azure.core.exceptions.HttpResponseError:`.

Can you also add a changelog? Otherwise this PR looks great, thanks so much for fixing this issue!!

Reply:
yes, changed to be just `:raises ~azure.core.exceptions.HttpResponseError:`  and added a changelog

Reply:
yes, changed to be just `:raises ~azure.core.exceptions.HttpResponseError:`  and added a changelog